README.md

```diff
@@ -1,7 +1,6 @@
 # Capybara
 
-[![Build Status](https://secure.travis-ci.org/teamcapybara/capybara.svg)](https://travis-ci.org/teamcapybara/capybara)
-[![Build Status](https://ci.appveyor.com/api/projects/status/github/teamcapybara/capybara?svg=true)](https://ci.appveyor.com/api/projects/github/teamcapybara/capybara)
+![Build Status](https://github.com/teamcapybara/capybara/actions/workflows/build.yml/badge.svg)
```

Comment:
This makes the badge link to the SVG file. Maybe it can link to https://github.com/teamcapybara/capybara/actions/workflows/build.yml instead so readers can get more information?

Reply:
That's a good idea, done! Thanks for the suggestion.